techniques/system/distributePolicy/1.0/rudder-rsyslog-root.st

```diff
@@ -70,3 +70,17 @@ $template RudderReportsFormat,"insert into RudderSysEvents (executionDate, nodeI
 
 ## Syslog messages from "rudder" forwarded from AIX
 :msg, ereregex, "from .*: rudder:" ~
+
+&if(SKIPIDENTIFY)&
+
+$AllowedSender UDP, &ALLOWEDNETWORK&
+$AllowedSender TCP, &ALLOWEDNETWORK&
```

Comment:
you ought to test (as a matter of fact, i know that it will output 
192.168.0.42/12192.168.42.0/31 (all texts stuck))